### DISCUSSION TITLE
Set Proxy

### DISCUSSION THREAD

**Discussion by Azadfalah** [tulir/whatsmeow#283] [General]
hi friends

Windows is set to this proxy 127.0.0.1:1080

how do I set this proxy for whatsmeow?

Thank you

**Comment by cfklkj** [comment]
like this:
	proxy := http.ProxyURL(&url.URL{
		Scheme: "socks5", // or http/https depending on your proxy
		Host:   "192.168.5.116:1081",
		User:   url.UserPassword("aaa", "bbb"),
		//Path:   "/",
	})
	client.SetProxy(proxy)
